markdown/3.x/en/faq.md

```diff
@@ -3,7 +3,21 @@
 #### Question:
 I want to use FACT-Finder Web Components with Angular, but data binding used in the components conflicts with Angular's `{{ }}` binding. Can you fix it?
 #### Answer:
-Default binding delimiters can be changed via `ff-communication`'s `mustache-delimiters` attribute. For more information please check the `Underlying Engine (Mustache)` section of the [Template Engine documentation](/documentation/3.x/template-engine).
+You can add Angular's `ngNonBindable` attribute to the parent element to prevent Angular from parsing double curly braces. If you have to keep both Angular and FACT-Finder WebComponents bindings within the same element, you can use: 
+```html
+<ng-container ngNonBindable>
+    {{FF Binding}}
+</ng-container>
+```
+The `<ng-container>` tag will not be rendered in the DOM.
+
+Alternatively, you can change the default FACT-Finder Web Components binding delimiters via `ff-communication`'s `mustache-delimiters` attribute. For more information please check the `Underlying Engine (Mustache)` section of the [Template Engine documentation](/documentation/3.x/template-engine). Please note that HTML binding will still require a single curly bracket, what will cause an Angular error. You can utilize `ngNonBindable` to handle this issue as well:
+```html
+<ng-container ngNonBindable>
+    {{{FF HTML Binding}}}
+</ng-container>
+```
+In this case, please use triple curly braces, no matter the custom delimiters you chose.
```

Comment:
Do I understand correctly: you change Mustache delimiters to custom ones, like `[` and you still have to use `{{{ }}}` when FF Web Component is nested in `<ng-container ngNonBindable></ng-container>`? Wasn't it that we had to use `[[{}]]` to bind HTML using custom delimiters, or was there a change in a meanwhile?

Reply:
You understand correctly, you have to use `{{{ }}}` in this case. Angular will complain about `[[{ }]]`